src/components/user/buttons/followButton.tsx

```diff
@@ -206,3 +206,103 @@ export const FollowButtonIcon = ({
     </>
   );
 };
+
+export const FollowButtonTab = ({
```

Comment:
Follow button tab, sounds like it indicates the tab component. Rename this component.